lib/segment/src/index/hnsw_index/hnsw.rs

```diff
@@ -1023,6 +1025,11 @@ impl<TGraphLinks: GraphLinks> VectorIndex for HNSWIndex<TGraphLinks> {
             .unwrap_or_else(|| self.graph.num_points())
     }
 
+    fn size_of_searchable_vectors_in_bytes(&self) -> Option<usize> {
+        // get the size from the vector storage
+        None
```

Comment:
Can the dense vector dense do the calculation?
Like call the function from dense storage which we know is defined?